src/parser/events-assets/grammar.ts

```diff
@@ -56,7 +55,7 @@ interface NearleyLexer {
   reset: (chunk: string, info: any) => void;
   next: () => NearleyToken | undefined;
   save: () => any;
-  formatError: (token: NearleyToken) => string;
+  formatError: (token: never) => string;
```

Comment:
question(non-blocking): What is the reason for changing the type to `never`?

Reply:
file was generated. I assume this was changed in the newer version of nearley for some reason.. not sure